tsconfig.integration.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This tsconfig file helps ensure that the integration codegen files produced valid TypeScript. I already found a bug in the current setup where `Variables` aren't imported from the types file, which this will catch.